main.js

```diff
@@ -0,0 +1,86 @@
+// let name = document.location.pathname.slice(1);
+if (!name) {
+  console.log("no place name specfied");
+} else {
+  console.log("fetching place named '" + name + "'");
+
+  fetch(name + ".json")
+    .then(function(response) {
+      return response.text();
+    })
+    .then(function(myText) {
+      console.log(myText);
+    });
+}
+
+function getRestaurantNames() {
+  fetch("all.json")
+    .then(response => response.json())
+    .then(object => {
+      console.log(object)
+    });
+};
+
+getRestaurantNames();
+
+function getWebsite(address) {
+  // fetch(name + ".json");
```

Comment:
Get rid of unused/empty functions to keep your code clean. If you're intending to build out this function later; make a note about what you intend to do with it.